slash/css/src/Layout/Container/Container.css

```
@@ -0,0 +1,23 @@
+.af-container {
+  margin: 0 30px;
+
+  @media (1016px <= width <= 1271px) {
+    margin: 0 40px;
+  }
+
+  @media (1272px <= width <= 1431px) {
+    margin: 0 56px;
+  }
+
+  @media (1432px <= width <= 1671px) {
+    margin: 0 56px;
+  }
+
+  @media (1672px <= width <= 1911px) {
+    margin: 0 56px;
+  }
```

C'est pour être raccord avec le reste du CSS que tu fais ça en 3 règles ? Car tu pourrais juste faire 
```suggestion
  @media (1272px <= width <= 1911px) {
    margin: 0 56px;
  }
```

---

De même, plutôt que d'être en pixel, vous n'êtes pas en _em_ ou _rem_ ?

---

oui c'est vrai qu'on pourrait faire un seul breakpoint, c'était pour suivre la liste qu'on a côté UX, mais si y'a pas de différence tu as raison

---

done pour les deux remarques